src/app/ui/button/button.ts

```diff
@@ -14,10 +14,15 @@ export class Button {
         const button = document.createElement('a');
         button.text = text;
         button.classList.add('gdg-raised-button');
+        button.classList.add('disabled');
```

Comment:
Button should not be automatically set to `disabled`. Add a `disable` method so it can be set to disabled from outside